mysite/__init__.py

```diff
@@ -0,0 +1 @@
+"""Django website package."""
```

Comment:
Please don't state obvious things. It's obvious that `mysite` is a package just because it has `__init__.py` in it. And the future reader looks right at this file. Stating that is no better than stating that you're reading this message right now.
Try to think of how to be useful to your future self.

Reply:
@RootenberG ^

Reply:
@webknjaz 

I don’t quite understand how to describe this module differently.

Reply:
You're describing a package. Not a module. But you shouldn't write a word package because it's obvious.

Reply:
@webknjaz 
```
"""Iooking for submodules inside this directory to import them"""

Reply:
`s/I/l/`

Reply:
No, you shouldn't describe how Python's internal import machinery works.

Reply:
What is `mysite` package?

Reply:
> What is `mysite` package?

It's directory is the actual Python package for project

Reply:
No. It's a Python package which means something in the scope of the project.

Reply:
> What is `mysite` package?



> No, you shouldn't describe how Python's internal import machinery works.

But __init.py__ does just that, I think this is not a bad description.

Reply:
Exactly. It describes how Python works, but it should describe what this package is.

Reply:
> Exactly. It describes how Python works, but it should describe what this package is.

Django website package?

Reply:
See https://github.com/kpi-web-guild/django-girls-blog-RootenberG/pull/22#discussion_r313629140

Reply:
@RootenberG yes, that would work. You can also be more specific by saying that it's a blog app. The framework detail is less important, though.